.eslintrc

```diff
@@ -3,8 +3,25 @@
     "mocha": true,
     "node": true
   },
-  "extends": "airbnb-base",
+  "extends": ["airbnb-base", "prettier"],
+  "overrides": [
+    {
+      "files": ["*.spec.js"],
+      "rules": {
+        "no-unused-expressions": "off"
+      }
+    }
+  ],
   "rules": {
-    "no-param-reassign": 0
+    "global-require": "off",
```

Comment:
to turn off rules about using `require` inside the functions, e.g. https://github.com/springuper/express-iroute/blob/master/lib/loader.js#L36

also for unit testing.

Reply:
but this could be adjusted when we switch module pattern to `esm`